src/test/java/uk/gov/hmcts/ethos/replacement/docmosis/helpers/Et3ResponseHelperTest.java

```diff
@@ -93,4 +101,54 @@ void givenStartDateInTheFutureAndEndDateBeforeStartDate_shouldReturnErrors() {
     }
+
+    @Test
+    void getDocumentRequest_buildsCorrectData() throws IOException, URISyntaxException {
+        Address et3Address = CaseDataBuilder.builder().createAddress("line1", "line2", "line3", "town", "county",
+            "postcode", "country");
+
+        caseData.setEt3RespondentAddress(et3Address);
+        caseData.setClaimant("claimant name");
+        caseData.setEt3ResponseRespondentLegalName("legal name");
+        caseData.setEt3ResponseRespondentCompanyNumber("1234");
+        caseData.setEt3ResponseRespondentEmployerType("employer type");
+        caseData.setEt3ResponseRespondentContactName("contact name");
+        caseData.setEt3ResponseEmploymentCount("100");
+        caseData.setEt3ResponseSiteEmploymentCount("100");
+        caseData.setEt3ResponseAcasAgreeReason("acas agree reason");
+        caseData.setEt3ResponseEmploymentStartDate("2022-01-01");
+        caseData.setEt3ResponseEmploymentEndDate("2022-01-02");
+        caseData.setEt3ResponseEmploymentInformation("employment information");
+        caseData.setEt3ResponseCorrectJobTitle("fall guy");
+        caseData.setEt3ResponseClaimantCorrectHours("168");
+        caseData.setEt3ResponsePayBeforeTax("69000");
+        caseData.setEt3ResponsePayTakehome("25000");
+        caseData.setEt3ResponseCorrectNoticeDetails("notice details");
+        caseData.setEt3ResponseContestClaimDetails("contest claim");
+        caseData.setEt3ResponseEmployerClaimDetails("ecc");
+        caseData.setEt3ResponseRespondentSupportDetails("support details");
+        caseData.setEt3ResponsePensionCorrectDetails("pension details");
+        caseData.setEt3ResponseHearingRespondent(Arrays.asList("Phone hearings"));
+        caseData.setEt3ResponseHearingRepresentative(Arrays.asList("Video hearings"));
+        caseData.setEt3ResponseRespondentPreferredTitle("Mr");
+        caseData.setEt3ResponseMultipleSites("Yes");
+        caseData.setEt3ResponseAcasAgree("No");
+        caseData.setEt3ResponseAreDatesCorrect("No");
+        caseData.setEt3ResponseIsJobTitleCorrect("No");
+        caseData.setEt3ResponseClaimantWeeklyHours("No");
+        caseData.setEt3ResponseEarningDetailsCorrect("No");
+        caseData.setEt3ResponseIsNoticeCorrect("Not applicable");
+        caseData.setEt3ResponseRespondentContestClaim("Yes");
+        caseData.setEt3ResponseRespondentSupportNeeded("Yes");
+        caseData.setEt3ResponseContinuingEmployment("No");
+        caseData.setEt3ResponseIsPensionCorrect("No");
+        caseData.setEt3ResponseContactPreference("Post");
+        caseData.setEt3ResponsePayFrequency("Weekly");
+
+        String expected = new String(Files.readAllBytes(Paths.get(Objects.requireNonNull(getClass().getClassLoader()
+            .getResource("et3ResponseDocument.json")).toURI())));
+
+        String actual = Et3ResponseHelper.getDocumentRequest(caseData, "any");
+        assertThat(actual, is(expected));
```

Comment:
Minor:  Better to do it like this from a readability perspective:
```
assertThat(actual)
    .isEqualTo(expected);

```

Reply:
Where does this `assertThat` come from? This file is currently importing from `org.hamcrest.MatcherAssert.assertThat` and it doesnt recognise an `assertThat` that takes a single argument